raccoongang/superset
# feat: [ZNO-2042] add analytical DB (clickhouse)

### SUMMARY
Added analytical database  (clickhouse)

### TESTING INSTRUCTIONS
- open path /databaseview/list/
- you should see the "clickhouse" database
- the database connection test must be successful

### ADDITIONAL INFORMATION
